examples/QmlBarcodeGenerator/QmlBarcodeGenerator.pro

```diff
@@ -5,6 +5,10 @@ CONFIG += c++17
 
 DEFINES += QT_DEPRECATED_WARNINGS
 
+#CONFIG += qmltypes
+#QML_IMPORT_NAME = com.scythestudio.scodes
+#QML_IMPORT_MAJOR_VERSION = 1
```

Comment:
Remove dead code - it won't be any longer needed if we don't use QML_ELEMENT macro to support Qt 5.15 CMake projects